doc/modules/feature_extraction.rst

```diff
@@ -100,6 +100,62 @@ of the time. So as to make the resulting data structure able to fit in
 memory the ``DictVectorizer`` class uses a ``scipy.sparse`` matrix by
 default instead of a ``numpy.ndarray``.
 
+.. _column_transformer:
+
+Columnar Data
+=============
+Many datasets contain features of different types, say text, floats and dates,
+where each type of feature requires separate preprocessing.
+Often it is easiest to preprocess data before applying scikit-learn methods, for example using
+pandas.
+If the preprocessing has parameters that you want to adjust within a
+grid-search, however, they need to be inside a transformer. This can be
+achieved very simply with the :class:`ColumnTransformer`.  The
+:class:`ColumnTransformer` works on pandas dataframe, dictionaries, and other
+objects that implement ``getattr`` so select a certain attribute or column.
+
+.. note::
+    :class:`ColumnTransformer` expects a very different data format from the numpy arrays usually used in scikit-learn.
+    For a numpy array ``X_array``, ``X_array[1]`` will give a single sample (``X_array[1].shape == (n_samples.)``), but all features.
```

Comment:
Yeah, I didn't update the rst docs yet, and I also saw there are still some errors like these